src/task/AttachMediaInputTask.ts

```diff
@@ -18,11 +18,7 @@ export default class AttachMediaInputTask extends BaseTask {
   async run(): Promise<void> {
     const transceiverController = this.context.transceiverController;
     transceiverController.setPeer(this.context.peer);
-    try {
-      transceiverController.setupLocalTransceivers();
-    } catch (error) {
-      throw error;
-    }
+    transceiverController.setupLocalTransceivers();
```

Comment:
so this would not throw any error ?

Reply:
The catch just rethrows it :P. My linter was complaining about it so figured i'd clean it up.

Reply:
I am blind :))